src/sentry/replays/endpoints/project_replay_recording_segment_details.py

```diff
@@ -0,0 +1,48 @@
+from django.http import StreamingHttpResponse
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.serializers import serialize
+from sentry.models import File
+from sentry.replays.models import ReplayRecordingSegment
+from sentry.replays.serializers import ReplayRecordingSegmentSerializer
+
+
+class ProjectReplayRecordingSegmentDetailsEndpoint(ProjectEndpoint):
```

Comment:
think you'll have to mark this endpoint as private with the member var `private = true`